### PR TITLE
add version tag to integrations docker images

### PR DESCRIPTION
**Pull request by cgardens**
## What
* we want to use a version if the image that we know works.
* right now, if you have an old version of any integration on your laptop, that old version will be used.

### CODE REVIEW

**Comment by michel-tricot** [comment]
for the time being we might force docker to ignore the cache, what do you think?

**Comment by cgardens** [comment]
@michel-tricot I'm fine with that. Are you suggesting that we do that instead of this PR or in addition?

File: dataline-integrations/singer/postgres/source/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.7-slim
 
-LABEL io.dataline.version=1.0.0
+LABEL io.dataline.version=0.1.1
```
Comment (michel-tricot) [inline dataline-integrations/singer/postgres/source/Dockerfile:3]: this should be 0.1.0 based on the Integration map

**Review by michel-tricot — APPROVED** [review]

**Review by jrhizor — APPROVED** [review]

File: dataline-integrations/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+### Upgrading an Integration
+1. Make sure you can build the integration locally
+
+    ```
+    ./tools/integrations/manage.sh build <path to integration>
```
Comment (sherifnada) [inline dataline-integrations/readme.md:5]: Should we remove the gradle `buildImage` tasks then?

**Review by sherifnada — APPROVED** [review]

File: dataline-integrations/readme.md (hunk 2 above)
Comment (michel-tricot) [reply dataline-integrations/readme.md:5]: nope, manage uses it

Comment (sherifnada) [reply dataline-integrations/readme.md:5]: nice

**Pull request merged**